packages/nativescript-stripe/standard/index.ios.ts

```diff
@@ -211,11 +211,15 @@ class StripePaymentDelegate extends NSObject implements STPPaymentContextDelegat
 	paymentContextDidCreatePaymentResultCompletion(paymentContext: STPPaymentContext, paymentResult: STPPaymentResult, completion: (p1: STPPaymentStatus, p2: NSError) => void): void {
 		StripeStandardConfig.shared.backendAPI
 			.capturePayment(paymentResult.paymentMethod.stripeId, paymentContext.paymentAmount, createShippingMethod(paymentContext), createAddress(paymentContext.shippingAddress))
-			.then((value) => {
-				completion(STPPaymentStatus.Success, null);
+			.then((value: any) => {
+                if(!value._native.lastPaymentError || value._native.lastPaymentError == "undefined") {
```

Comment:
@kefahB @triniwiz I'm having trouble with this change. Every payment fails with the error originating from this line:
`Cannot read property 'lastPaymentError' of undefined`

In my case, the variable "value" is the result of the promise returned from capturePayment defined in my own code. This is referring to a _native property which my promise doesn't return. Seems like this line of code is supposed to call something else internal to the Stripe pod to get the lastPaymentError?